test/tests/dialog.js

```diff
@@ -246,7 +249,7 @@ ariaTest('tab changes focus within dialog', exampleFile, 'key-tab', async (t) =>
     t.true(
       await checkFocus(t, ex.dialog2FocusableEls[i]),
       'Focus should be on "' + ex.dialog2FocusableEls[i] + '" after ' +
-        (i + 1) + ' tabs have been sent to dialog 2'
+        (i + 2) + ' tabs have been sent to dialog 2'
```

Comment:
What does `2` represent here? A comment to explain these offsets will be helpful.

Reply:
it's the number of times tab has been sent to the browser. The for-loop is index 0, but we say that we sent "1" tab to the browser, not "0" tabs. in the case where there is 2, there is a tab send before the for loop starts.

Lemme think about how to make this more clear..

Reply:
oh wow! I thought I had the logic wrong the FIRST time but I actually had it wrong THIS time.